scikeras/wrappers.py

```diff
@@ -64,10 +65,66 @@ class BaseWrapper(BaseEstimator):
         "multioutput": True,
     }
 
+    _fit_params = {
+        # parameters destined to keras.Model.fit
+        "callbacks",
+        "batch_size",
+        "epochs",
+        "verbose",
+        "callbacks",
+        "validation_split",
+        "shuffle",
+        "class_weight",
+        "sample_weight",
+        "initial_epoch",
+        "validation_steps",
+        "validation_batch_size",
+        "validation_freq",
+    }
+
+    _predict_params = {
+        # parameters destined to keras.Model.predict
+        "batch_size",
+        "verbose",
+        "callbacks",
+        "steps",
+    }
+
+    _compile_params = {
+        # parameters destined to keras.Model.compile
+        "optimizer",
+        "loss",
+        "metrics",
+        "loss_weights",
+        "weighted_metrics",
+        "run_eagerly",
+    }
+
+    _wrapper_params = {
+        # parameters consumed by the wrappers themselves
+        "warm_start",
+        "random_state",
+    }
```

Comment:
There's some testing to make sure that these are a subset of keras.Model's parameters. In the future, once we've figured out the usage of `class_weight` and others, we should add tests that also check that the default initializer for the wrappers also accepts all of these (except for `sample_weight` and such).